### ISSUE TITLE
Transfigure image failed to run due to bazel version mismatch

### ISSUE THREAD

**Issue by chaodaiG** [kubernetes/test-infra#17997]
**What happened**:
Running transfigure with latest image failed due to:
```
Running kubernetes/test-infra tests...
ERROR: The project you're trying to build requires Bazel 3.0.0 (specified in /home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/test-infra/.bazelversion), but it wasn't found in /usr/local/lib/bazel/bin.
```

Link: https://oss-prow.knative.dev/view/gs/oss-prow/pr-logs/pull/GoogleCloudPlatform_oss-test-infra/395/pull-oss-test-infra-check-testgrid-config/1273664592261681152

**What you expected to happen**:
Transfigure run as expected

**How to reproduce it (as minimally and precisely as possible)**:

**Please provide links to example occurrences, if any**:

**Anything else we need to know?**:

**Comment by BenTheElder** [comment]
Is this still the case?

**Comment by chaodaiG** [comment]
It's fixed by #17999 
/close
